oan-model/src/test/java/org/jax/oan/core/MedicalActionTargetExtendedTest.java

```diff
@@ -0,0 +1,28 @@
+package org.jax.oan.core;
+
+import org.junit.jupiter.api.Test;
+import org.monarchinitiative.phenol.ontology.data.TermId;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MedicalActionTargetExtendedTest {
+
+	@Test
+	void construct() {
+		TermId termId = TermId.of("MAXO:1234");
```

Comment:
These seem to be 'does it work' tests. They are something but more useful is 'can I make it fail' tests.